backend/amp-live-list.go

```diff
@@ -157,13 +165,71 @@ func createMetadata(r *http.Request) []BlogPosting {
 	return result
 }
 
-func createLiveBlogSample(newStatus int, timestamp time.Time, r *http.Request) LiveBlogSample {
+func createLiveBlogSample(newStatus int, timestamp time.Time, r *http.Request, page Page) LiveBlogSample {
 	if newStatus > len(blogs) {
 		newStatus = len(blogs)
 	}
 	blogItems := getBlogEntries(newStatus, timestamp)
 	score := createScore(newStatus, 0)
-	return LiveBlogSample{BlogItems: blogItems, FootballScore: score, BlogMetadata: createMetadata(r)}
+	firstBlogID := strings.TrimPrefix(r.URL.Query().Get("from"), BLOG_ID_PREFIX)
```

Comment:
perform this `strings.TrimPrefix(r.URL.Query().Get("from")` outside this method. Creating a blog entry should not depend on the request.